content/blog/2022/12/kubewarden-2022-wrapped.md

```diff
@@ -0,0 +1,120 @@
+- Flavio Castelli
+date: 2022-12-29
+---
+
+With 2022 almost over, it's time to look back at what happened within
+the Kubewarden project during the last year.
+
+## The 1.0 release
+
+A significant milestone for the project in 2022 was
+the [release of  Kubewarden v1.0.0](/blog/2022/06/v1-release/) during the month of June.
+
+With this release, the Kubewarden team committed to the stability of all the
+public interfaces of the project and all its Kubernetes Custom Resource Definitions.
+
+Moreover, the project was considered ready to be used in production environments.
+
+## CNCF inclusion
+
+Another exciting event took place during the month of June. That's when
+[Kubewarden's inclusion in the CNCF sandbox]((/blog/2022/06/cncf-sandbox-inclusion/)
+took place.
+
+On top of being a great recognition of the quality and potential of the project, the
+acceptance into CNCF led to an exciting integration with
+[ArtifactHub](/blog/2022/07/artifact-hub-supports-kubewarden/).
+This proved to be a great improvement not just for policy authors, but also
+for the all Kubewarden's users who can now find all the existing policies
+on the [familiar ArtifactHub interface](https://artifacthub.io/packages/search?kind=13&sort=relevance&page=1).
+
+By being part of CNCF, Kubewarden also got access to another CNCF project,
+[CLOMonitor](https://clomonitor.io/projects/cncf/kubewarden).
+Thanks to CLOMonitor, Kubewarden's users have a clear picture of the project health
+and can assess its development and security practices.
+
+## Pod Security Policy replacement
+
+During the month of August, Kubernetes 1.25 release finally removed
+[Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/).
+
+This event was planned and advertised for a long time. Due to this,
+the Kubewarden developers were able to provide
+[tooling, policies, and documentation](https://docs.kubewarden.io/tasksDir/psp-migration)
+to smoothen the migration process from Kubernetes Pod Security Policies to
+Kubewarden policies.
+
+## Secure supply chain
+
+The topic of secure supply chain started to emerge near the end of 2021, but it
+was during this year that many projects and organizations tackled it.
+
+The [sigstore](https://sigstore.dev) project proved to be one of the pillars of
+these initiatives.
+Kubewarden invested early in the Sigstore project, with some of our maintainers being
+active also within the Sigstore community.
+
+During the last year Kubewarden embraced Sigstore to
+[securely distribute its own assets](/blog/2022/04/securing-kubewarden-policies/),
+like container images, policies, helm charts and cli utilities.
+Moreover, we exposed Sigstore capabilities to our policy authors and provided
+a [ready to use policy](https://artifacthub.io/packages/kubewarden/verify-image-signatures/verify-image-signatures)
+that can be used to vet the container images used by Kubernetes workloads.
+
+Sigstore provides different ways to sign artifacts: with
```

Comment:
I think we should say the sigstore integration here instead of just Sigstore.
```suggestion
The Sigstore integration provides Kubewarden users different ways to sign artifacts: with
```